client/src/App.js

```diff
@@ -35,15 +36,17 @@ function App() {
           // retrieve validUrl from response
           validUrl = proxyUrlResJson.validUrl;
           setValidUrl(validUrl);
+          // retrieve pageDimensions from response (use for downloading)
+          setPageDimensions(proxyUrlResJson.pageDimensions);
         } catch {
-          console.error('Error in fetching page dimensions. Using default dimensions.');
+          console.error('Error in calling `/pdftron-proxy`. Check server log');
         }
         const { pathname } = new URL(validUrl);
 
         setResponse({
           url: `${PATH}${pathname}`,
           thumb: '',
-          ...pageDimensions,
+          ...defaultPageDimensions,
```

Comment:
Can you add a comment why `defaultPageDimensions` is being passed to the package and not pageDimensions. It is not clear from reading the code. `pageDimensions` might be better renamed to something like `pageDimensionsForDownload`. This would help with readability.